charts/kube-prometheus-stack/values.yaml

```diff
@@ -2372,6 +2373,63 @@ prometheus:
     ## port to 2379 and allow etcd scraping provided it is running on all Kubernetes master nodes
     ##
     additionalScrapeConfigs: []
+      # ipv6:
+      #   - job_name: 'kubernetes-pods'
+      #     kubernetes_sd_configs:
+      #       - role: pod
+      #     relabel_configs:
+      #     - action: keep
+      #       regex: 'true;(\d+)'
+      #       source_labels:
+      #       - __meta_kubernetes_pod_annotation_prometheus_io_scrape
+      #       - __meta_kubernetes_pod_annotation_prometheus_io_port
+      #     - action: replace
+      #       regex: (.+)
+      #       source_labels:
+      #       - __meta_kubernetes_pod_annotation_prometheus_io_path
+      #       target_label: __metrics_path__
+      #     - action: replace
+      #       regex: '(.*);(\d+)'
+      #       replacement: '[$1]:$2'
```

Comment:
I don't know exactly how dual stack integrates with prometheus, but it seems like we may prefer a regex that matches IPv4 OR IPv6.

I did try to write one, but it was pretty hard. I do suspect you could have multiple `action`s for the same `__address__` target though

Reply:
Unfortunately, it's not only the regex, it's also the replacement (line 2393) that was wrapped by []. Not sure we can do this for both IPv4 & IPv6 using the same regex.